Makefile

```diff
@@ -181,8 +181,11 @@ deploy-operator-on-ocp:
 olm-tests:
 	./hack/ci/run-olm-tests.sh
 
+update-dependencies:
+	go get github.com/kiegroup/kogito-operator@master
```

Comment:
This is too master specific. If we create a release branch, then this may create some problems with this command ...
I guess we should have the branch as global var, like we have with the operator version and we should update depending on which branch we are or which branch is set as parameter.

Reply:
+1, we probably should use a branch or commit id to version the imports.

Reply:
I added the branch as the global variable `UPSTREAM_VERSION`. I considered using an argument for the branch, but since I'm adding `update-dependencies` to the `before-pr` command, I don't think that would work.